app/views/user_mailer/claim_notification.html.erb

```diff
@@ -20,17 +20,13 @@
     <% end %>
   </ul>
 
-  <p>To preserve rec lists and bookmarks, the imported archive's web addresses may redirect to the imported copy of these works for a limited time (check the announcement post for your archive to be sure). If you've already uploaded a copy of these works and you did NOT use the import from URL feature, there will be two copies of the same work on the archive. </p>
+  <p><%= t '.html.part6' %></p>
 
-  <p>If you would like Open Doors to update the redirect to point to your pre-existing work, please delete the imported copy, and <%= style_link('contact Open Doors', 'http://transformativeworks.org/contact/open%20doors') %> with your AO3 account name, your account name on the imported archive, and the title and URL of the fanwork you would like the redirect to point to. (If you have multiple works you would like to change the redirects for, you can list these in one email.)</p>
+  <p><%= t('.html.part7', open_doors: style_link(t('.contact_open_doors'), 'http://transformativeworks.org/contact/open%20doors')).html_safe %></p>
 
-  <p>If you had other works on the imported archive under an e-mail address you can no longer access, please <%= style_link('contact Open Doors', 'http://transformativeworks.org/contact/open%20doors') %> with any information that can help verify your identity.</p>
+  <p><%= t('.html.part8', open_doors: style_link(t('.contact_open_doors'), 'http://transformativeworks.org/contact/open%20doors')).html_safe %></p>
 
-  <p>For other inquiries, please <%= support_link('contact Support') %>.</p>
+  <p><%= t('.html.part9', support: support_link(t '.support')).html_safe %></p>
 
-  <p>If you're contacting us, please whitelist email addresses from @transformativeworks.org and check your spam folders for our reply.</p>
-
-  <p>Best,<br/>
-    The Open Doors team<br/>
-    Organization for Transformative Works</p>
-<% end %>
\ No newline at end of file
+  <%= t '.html.part10' %> 
```

Comment:
Translating the email will never require us to flip the order of the final paragraph and the close, so let's leave them separate, i.e.:
```
<p><%= t '.html.part10' %></p>

<p><%= t '.html.closing' %></p>
```